### PR TITLE
HTTP content-range 

### PR DESCRIPTION
**Pull request by guoliang-gst**
#### Motivation
Make the parameter description more rigorous from the perspective of computer science.
range-start: begins from 0
range-end: max number is total length - 1

### CODE REVIEW

File: files/en-us/web/http/headers/content-range/index.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ Content-Range: <unit> */<size>
 - \<unit>
   - : The unit in which ranges are specified. This is usually `bytes`.
 - \<range-start>
-  - : An integer in the given unit indicating the beginning of the request range.
+  - : An integer in the given unit indicating the beginning unit position of the request range.
```
Comment (hamishwillee) [inline files/en-us/web/http/headers/content-range/index.md:54]: I tried an internet search to confirm the term "unit position" means what I think it does. Couldn't find a good match,  implies that many readers will not find this more any helpful than the current text. Might we just spell it out?
```suggestion
  - : An integer in the given unit indicating the start of the request range (zero-indexed).
```
Same for the other case.

**Review by hamishwillee — APPROVED** [review]
Thanks very much - that is more precise and I think clear.

File: files/en-us/web/http/headers/content-range/index.md (hunk 1 above)
Comment (EddyZhang38) [reply files/en-us/web/http/headers/content-range/index.md]: Thanks for your recommendations! I have updated the PR. 
I searched a doc for your reference: https://datatracker.ietf.org/doc/html/rfc7233#section-2.1

**Comment by EddyZhang38** [comment]
Updated the PR. 
>  The first-byte-pos value in a byte-range-spec gives the byte-offset
   of the first byte in a range.  The last-byte-pos value gives the
   byte-offset of the last byte in the range; that is, the byte
   positions specified are inclusive.  Byte offsets start at zero.

 Ref: https://datatracker.ietf.org/doc/html/rfc7233#section-2.1

**Pull request merged**